inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/detection_output/detection_output_kernel_ref.cpp

```diff
@@ -123,7 +123,7 @@ DetectionOutputKernelRef::DispatchData SetDefault(const detection_output_params&
             dispatchData.lws = {1, 1, 1};
         } else {
             dispatchData.gws = {input.Batch().v, 1, 1};
-            dispatchData.lws = {input.Batch().v, 1, 1};
+            dispatchData.lws = {1, 1, 1};
```

Comment:
Can LWS downgrade has potential impact on performance for cases with big batches?

Reply:
As a result of checking performance impact, The latency for final stage improved by changing to work-groups as many as batches rather than work-items as many as batches in single work-group.